### PR TITLE
PRIME-1775 Fix TOA drop-down in Paper Enrollees

### PR DESCRIPTION
**Pull request by AMiKaH**

### CODE REVIEW

File: prime-angular-frontend/src/app/shared/enums/agreement-type.enum.ts [hunk 1]
```diff
@@ -7,11 +14,28 @@ export enum AgreementType {
...
+
+// Paper Enrolment
+export enum AgreementTypePaperEnrolment {
+  COMMUNITY_PHARMACIST_TOA = 1,
+  REGULATED_USER_TOA,
+  OBO_TOA,
+  PHARMACY_OBO_TOA
+}
+
+export const AgreementTypeNameMapPaperEnrolment: Record<AgreementTypePaperEnrolment, string> = {
```
Comment (hogmann) [inline prime-angular-frontend/src/app/shared/enums/agreement-type.enum.ts:35]: I know this isn't really defined anywhere, but probably want to put the description of the variable as a Prefix with the type of the object being the suffix.

So like PaperEnrolmentAgreementTypeNameMap

Comment (AMiKaH) [reply prime-angular-frontend/src/app/shared/enums/agreement-type.enum.ts:35]: Got it, thanks!

**Review by hogmann — APPROVED** [review]
LGTM

**Review by Paahn — APPROVED** [review]
thumbs up slowly

**Pull request merged**